.github/workflows/release.yaml

```diff
@@ -22,69 +22,51 @@ jobs:
 
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
+      - name: Check if any changes since last tag
+        id: check
+        run: |
+          git fetch --tags
+          if [ -z "$(git tag --points-at HEAD)" ]; then
+            echo "Nothing points at HEAD, so we need a new tag+release."
+            echo "need_release=yes" >> $GITHUB_OUTPUT
+          else
+            echo "A tag already points to head, no need for a new tag+release."
+            echo "need_release=no" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Bump version and push tag
+        uses: mathieudutour/github-tag-action@a22cf08638b34d5badda920f9daf6e72c477b07b # v6.2
```

Comment:
how hard would it be to do this ourselves? 
prefer not using other actions that we need to pass our GITHUB token to

Reply:
I can fix that, just need a time

Reply:
Cool — yeah could we come back to this? This isn't a net change from this PR, this PR just moves it to this file

Reply:
definitely not blocking